Tasks/CloudFormationExecuteChangeSet/helpers/ExecuteChangeSetTaskParameters.ts

```diff
@@ -10,16 +10,25 @@ import tl = require('vsts-task-lib/task');
 import { AWSTaskParametersBase } from 'sdkutils/awsTaskParametersBase';
 
 export class TaskParameters extends AWSTaskParametersBase {
+
+    public static readonly ignoreStackOutputs: string = 'ignore';
```

Comment:
See above--consider converting these to top-level exported constants.

Reply:
We can think about that when this is refactored, please put that feedback on this PR: https://github.com/aws/aws-vsts-tools/pull/168